mobile-widgets/qml/mobile-resources.qrc

```diff
@@ -41,6 +41,9 @@
 		<file alias="mapwidget-toggle-street.png">../../map-widget/qml/icons/mapwidget-toggle-street.png</file>
 		<file alias="mapwidget-zoom-in.png">../../map-widget/qml/icons/mapwidget-zoom-in.png</file>
 		<file alias="mapwidget-zoom-out.png">../../map-widget/qml/icons/mapwidget-zoom-out.png</file>
+        <file alias="dive-location-marker-icon">../../map-widget/qml/icons/mapwidget-marker.png</file>
+        <file alias="dive-location-marker-inactive-icon">../../map-widget/qml/icons/mapwidget-marker-gray.png</file>
+        <file alias="dive-location-marker-selected-icon">../../map-widget/qml/icons/mapwidget-marker-selected.png</file>
```

Comment:
indentation issues. but this also doubles some aliases for the same file names.
see `mapwidget-marker*` above.

i think this commit 8ef87e618a was a bad idea, @dirkhh .
@janiversen the mobile and desktop version need to use the same QRC aliases as those in the QML for the map widget - MapWidget.qml.

given this build from a couple of days ago just worked fine on my Android device:
https://github.com/Subsurface-divelog/subsurface/releases/tag/continuous

my only guess is that we are using desktop resources for the Android mobile build. o_O.

Reply:
you are corrct, put this PR on hold, but at least I know what the problem. Will continue work tomorrow.

Reply:
@neolit123 just checked mapwidget/qml should have a qrc file instead of having defintions in both subsurface.qrc and mobile-widgets.qrc. I need some hours sleep but then I will update this PR.

Reply:
sounds good.

Reply:
I like the idea of one qrc file that's shared between mobile and desktop for the map
Also, @janiversen when you do a commit that fixes an issue, if you add a line like this in the commit message itself (not the PR text, the commit message)
```
Fixes #1410
```
Then GitHub closes the issue automagically.
Finally... please have a Subsurface config for your editor / IDE so that by default you don't mangle whitespace? It must be frustrating for you to always be reminded by the rest of us...

Reply:
thanks, I have my editors configured, but copy/paste on a mac does strange things.